app/views/rent_mailer/new_rent_send.erb

```diff
@@ -1,10 +1,10 @@
 <div>
   <ul>
-    <li>Fecha actual: <%= @rent.created_at %></li>
-    <li>Título libro: <%= @rent.book.title %></li>
-    <li>Año libro: <%= @rent.book.year %></li>
-    <li>Editor libro: <%= @rent.book.publisher %></li>
-    <li>Fecha inicio renta: <%= @rent.start_date %></li>
-    <li>Fecha término renta: <%= @rent.end_date %></li>
+    <li><%= t :created_at %>: <%= @rent.created_at %></li>
```

Comment:
Perfect! Another way is to pass the  variables  to the translations, some docs [here](https://edgeguides.rubyonrails.org/i18n.html#passing-variables-to-translationsl)

Reply:
Interesting :D